astroquery/jplhorizons/core.py

```diff
@@ -15,6 +12,9 @@
 from astropy.time import Time
 from astropy.utils.exceptions import AstropyDeprecationWarning
 from astropy.utils.decorators import deprecated_renamed_argument, deprecated_attribute
+from numpy import nan
```

Comment:
if you need to reorder imports, keep the numpy ones before astropy.

Reply:
copy that.